db/internal_stats.h

```diff
@@ -235,6 +281,19 @@ class InternalStats {
       this->num_input_records -= c.num_input_records;
       this->num_dropped_records -= c.num_dropped_records;
       this->count -= c.count;
+      int numOfReasons = static_cast<int>(CompactionReason::kNumOfReasons);
```

Comment:
naming convention is lower-case underscore-separated for local variables (we mostly try to follow google style guide).

Reply:
Done.